source/integrations/generate-lets-encrypt-certificate-using-certbot-for-minio.md

```diff
@@ -0,0 +1,72 @@
+# Generate Let's Encrypt certificate using Certbot for MinIO [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io)
```

Comment:
Do we need a Slack link in the page title?
Also, add a line break after the title. Uh, please.